packages/mdc-select/adapter.ts

```diff
@@ -148,7 +170,7 @@ export interface MDCSelectAdapter {
   /**
    * Gets the text content of the menu item element at the given index.
    */
-  getMenuItemTextAtIndex(index: number): string;
+  getMenuItemTextAtIndex(index: number): string | null;
```

Comment:
`getMenuItemTextAtIndex` should always return `string` type.

Reply:
I think we can typecast it with `... as string` operator in adapter implementation.